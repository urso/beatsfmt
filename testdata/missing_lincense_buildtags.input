// +build test
package abc

func def() {}
