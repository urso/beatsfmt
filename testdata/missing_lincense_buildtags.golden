// License header line 1
// License header line 2

// +build test
package abc

func def() {}
